src/components/alert/alert-component.ts

```diff
@@ -128,6 +140,29 @@ export class AlertCmp {
     }
   }
 
+  ionViewPreLoad() {
+    if (this.d.subview && this.d.subview.component) {
```

Comment:
I have noticed we have a lot of references to `this.d`. can we add a new const variable at the root, like:
```
const data = this.d;
```

then use data, instead of `this.d`.

Noticed that minifies like uglify can rename local variables like "data", but they can't with "instance" variables.

Also, it looks cleaner and local variable accesses are cheaper.